README.md

```diff
@@ -31,6 +28,16 @@ critical things like tuning performance and getting the most out of
 Spanner. Expect that you'll need to tweak and enhance what HarbourBridge
 produces.
 
+## Data Migration
+
+HarbourBridge supports two types of data migrations:
+
+* Bulk Migration -  In bulk migration HarbourBridge takes a snapshot of the data at the moment it starts migration and then creates a database with that data in Cloud Spanner.
+Changes which happen to the source database during the bulk migration are not written to Spanner. It is suggested for
+migrating moderate-size datasets to Spanner(up to about 100GB).
+
+* Streaming migration - A streaming migration consists of two components, a consistent snapshot of all the data in the database, and the stream of changes (writes and updates) since that snapshot, referred to as change database capture (CDC). Using HarbourBridge, the entire process can be orchestrated using a unified interface. It is suggested for larger databases(> 100GB).
```

Comment:
We don't do snapshots. We overwrite the changes based on timestamp.

This recommendation is not only for large databases, it is for any database that requires low downtime.